### DISCUSSION TITLE
.vtt Subtitle support for MediaElement iOS and Android

### DISCUSSION THREAD

**Discussion by 94mc** [CommunityToolkit/Maui#1018] [New Feature Discussions]
Hi , i am wondering if it is possible to implement closed captions to a mediaelement video. This is a business critical feature for myself and i cant seem to find any info on it in regards iOS in particular. Any thoughts or updates on this would be of great value

Thanks!

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
